src/transformers/expect-js.test.js

```diff
@@ -0,0 +1,291 @@
+      expect(stuff).toBeLessThan(42);
+      expect(stuff).toBeGreaterThan(42);
+      expect(stuff).toBeGreaterThan(42);
+    });
+    `
+);
+
+testChanged(
+    'maps expect throw matchers',
+    `
+    import expect from 'expect.js';
+
+    test(() => {
+      expect(stuff).to.throwError(Error);
+      expect(stuff).to.throwError();
+      expect(stuff).to.throw(Error);
+      expect(stuff).to.throw();
+      expect(fn).to.throwException(function (e) {
+        expect(e).to.be.a(SyntaxError);
+      });
+    });
+    `,
+    `
+    test(() => {
+      expect(stuff).toThrowError(Error);
+      expect(stuff).toThrow();
+      expect(stuff).toThrowError(Error);
+      expect(stuff).toThrow();
+      try {
+        fn();
+        fail();
+      } catch (e) {
+        expect(e).toBeInstanceOf(SyntaxError);
+      };
+    });
+    `
+);
+
+testChanged(
+    'maps expect fail matchers',
+    `
+    import expect from 'expect.js';
+
+    test(() => {
+      expect().fail('message');
+    });
+    `,
+    `
+    test(() => {
+      fail('message');
+    });
+    `
+);
+
+testChanged(
+    'maps expect a and an matchers',
+    `
+    import expect from 'expect.js';
+
+    test(() => {
+      expect(stuff).to.be.a('function');
+      expect(stuff).to.be.a('string');
+      expect(stuff).to.be.a('object');
+      expect(stuff).to.be.a('array');
+      expect(stuff).to.be.a('boolean');
+      expect(stuff).to.be.a('symbol');
+      expect(stuff).to.be.a('number');
+      expect(stuff).to.be.a(Thing);
+
+      expect(stuff).to.be.an('function');
+      expect(stuff).to.be.an('string');
+      expect(stuff).to.be.an('object');
+      expect(stuff).to.be.an('array');
+      expect(stuff).to.be.an('boolean');
+      expect(stuff).to.be.an('symbol');
+      expect(stuff).to.be.an('number');
+      expect(stuff).to.be.an(Thing);
+    });
+    `,
+    `
+    test(() => {
+      expect(typeof stuff).toBe('function');
+      expect(typeof stuff).toBe('string');
+      expect(typeof stuff).toBe('object');
+      expect(stuff).toBeInstanceOf(Array);
+      expect(typeof stuff).toBe('boolean');
+      expect(typeof stuff).toBe('symbol');
+      expect(typeof stuff).toBe('number');
+      expect(stuff).toBeInstanceOf(Thing);
+
+      expect(typeof stuff).toBe('function');
+      expect(typeof stuff).toBe('string');
+      expect(typeof stuff).toBe('object');
+      expect(stuff).toBeInstanceOf(Array);
+      expect(typeof stuff).toBe('boolean');
+      expect(typeof stuff).toBe('symbol');
+      expect(typeof stuff).toBe('number');
+      expect(stuff).toBeInstanceOf(Thing);
+    });
+    `
+);
+
+testChanged(
+    'maps expect within matchers',
+    `
+    import expect from 'expect.js';
+
+    test(() => {
+      expect(stuff).to.be.within(0, 100);
+    });
+    `,
+    `
+    test(() => {
+      expect(stuff > 0 && stuff < 100).toBeTruthy();
+    });
+    `
+);
+
+testChanged(
+    'standaloneMode: keeps expect import',
+    `
+    import exp from 'expect';
```

Comment:
I guess this should be `import exp from 'expect.js';` that is converted to `import expect from 'expect';`